src/app/base/components/bricks-integrator/bricks-integrator.component.ts

```diff
@@ -363,6 +368,7 @@ export class BricksIntegratorComponent implements OnInit, OnDestroy {
         else {
           this.config.api.data = c;
           this.config.api.data.data.attributes.partOfGroup = JSON.parse(c.data.attributes.partOfGroup);
+          this.config.api.data.data.attributes.partOfGroup = filterArrayMinusExcluded(this.config.api.data.data.attributes.partOfGroup, REMOVE_GROUP_NODES);
           this.config.api.data.data.attributes.partOfGroupText = this.config.api.data.data.attributes.partOfGroup.map(x => this.getGroupName(x)).join(", ");
           this.config.api.data.data.attributes.availableFor = JSON.parse(c.data.attributes.availableFor);
           this.config.api.data.data.attributes.integratorInputs = JSON.parse(c.data.attributes.integratorInputs);
```

Comment:
This could maybe outsourced in a method which has a name that shows what this else case is about. So I see that this else case happens if integratorInputs are not set (at least thats what I undestand) but I don't know straight up what then happens in the else case. Is it more of an error handling or setting of default values or something in this direction? I think the implicit knowledge which is hidden here in the else case could be expressed explicitly for the reader in a method name. But this is again maybe more of code style and could be applied to mulitple places in our app, so again, you have to decide :)

Reply:
I understand, then maybe add a comment at the top of the else-case body, with a short description of what is happening there ?

Reply:
Yes sure I will add a comment